### PR TITLE
reconstruct: Support object notation instead of xliff

### PR DESCRIPTION
**Pull request by AnyaTyu**
#7 @tadatuta

### CODE REVIEW

File: lib/reconstruct.js [hunk 1]
```diff
@@ -1,17 +1,29 @@
 var xamel = require('xamel'),
-    tag = process.env.USE_SOURCE ? 'source' : 'target';
+    tag = process.env.USE_SOURCE ? 'source' : 'target',
```
Comment (tadatuta) [inline lib/reconstruct.js]: let's switch 2 and 3 lines to group both requires together

File: lib/reconstruct.js [hunk 2]
```diff
@@ -1,17 +1,29 @@
...
-                text = unit.find(tag).text(true).join('');
+            units.forEach(function(unit) {
+                var id = unit.attr('id'),
+                    text = unit.find(tag).text(true).join('');
+
+                skeleton = skeleton.replace('%%%' + id + '%%%', text);
+            });
+
+            cb(null, skeleton);
+        });
```
Comment (tadatuta) [inline lib/reconstruct.js]: add `return` on the next line to get rid of `else` and extra indentation

File: lib/reconstruct.js [hunk 3]
```diff
@@ -1,17 +1,29 @@
...
+    } else {
+        xliff.forEach(function (unit) {
+            var id = unit.id,
+                text = strip(unit.target.content);
 
             skeleton = skeleton.replace('%%%' + id + '%%%', text);
         });
```
Comment (tadatuta) [inline lib/reconstruct.js:13]:
``` js
cb && cb(null, skeleton);
```

**Comment by AnyaTyu** [comment]
@tadatuta done

**Comment by tadatuta** [comment]
@AnyaTyu thank you!

**Pull request merged**